src/spaces/ictcp.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-unused-vars */
```

Comment:
Where are the unused vars here?

Reply:
`rec2020`, but I think the intent of the TODO is pretty clear.

Reply:
I don’t see a TODO?
I think we should just remove or comment out that line, @svgeesus what do you think?